docs/windows.md

```diff
@@ -0,0 +1,226 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay
+network to forward packets between nodes. Currently VXLAN and STT tunnels are
+supported.
+
+This page shows how to install Antrea Agent on Windows Nodes and register the
+Node to existing kubernetes cluster.
+
+For the detailed design of how Antrea Agent works on Windows, please refer to
+the doc [Antrea on Windows](https://docs.google.com/document/d/1lSis0XnKz8UcJSkxTgRtDhP2DAwQtcZDjT6ZRySUb48).
+
+### Components that run on Windows
+
+The following components should be configured and run on the Windows Node.
+* [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
+* OVS daemons
+* Antrea Agent
+* Kube-proxy
+
+Antrea-Agent and Kube-proxy run as processes on host and are managed by
```

Comment:
Antrea-Agent and Kube-proxy looks a little strange, I think either "Antrea agent" or "antrea-agent". Not common to capitalize program name such as "antrea-agent", "kube-proxy".

Reply:
OK. Use `antrea-agent` and `kube-proxy` instead of other names.

Done.